eclass/cvs.eclass

```diff
@@ -294,50 +295,61 @@ cvs_fetch() {
 
 	# Switch servers automagically if needed
 	if [[ ${mode} == "update" ]] ; then
-		cd "/${ECVS_TOP_DIR}/${ECVS_LOCALNAME}"
-		local oldserver=$(cat CVS/Root)
+		cd "/${ECVS_TOP_DIR}/${ECVS_LOCALNAME}" || die
+		local oldserver
+		oldserver=$(cat CVS/Root) || die
 		if [[ ${server} != "${oldserver}" ]] ; then
 			einfo "Changing the CVS server from ${oldserver} to ${server}:"
 			debug-print "${FUNCNAME}: Changing the CVS server from ${oldserver} to ${server}:"
 
 			einfo "Searching for CVS directories ..."
-			local cvsdirs=$(find . -iname CVS -print)
+			local cvsdirs
+			cvsdirs=$(find . -iname CVS -print) || die
```

Comment:
You could also move the die into the subshell, right?

Reply:
Doesn't make any difference, the subshell will return the status of the last command.

Reply:
The point is that you split this into two lines afaict just to avoid `die` seeing local instead.

Reply:
Done.